src/elements/header/header/readme.md

```diff
@@ -117,9 +130,33 @@ The result can be seen in the [home](/story/pages-home--home) and [home-logged-i
   .sbb-header-spacer {
     display: none;
   }
+
+  .sbb-header-spacer-logo {
+    display: block;
+  }
 }
 ```
 
+```html
+<sbb-header>
+  <sbb-header-button icon-name="..." expand-from="small"> ... </sbb-header-button>
+
+  <!-- Will be hidden on small screen sizes -->
+  <div class="sbb-header-spacer"></div>
+
+  <sbb-header-button icon-name="..."> ... </sbb-header-button>
+  <sbb-header-button icon-name="..."> ... </sbb-header-button>
+  <sbb-header-button icon-name="..." class="last-element"> ... </sbb-header-button>
+
+  <!-- Will only be shown on small screen sizes -->
+  <div class="sbb-header-spacer sbb-header-spacer-logo"></div>
```

Comment:
We don't need the header spacer here, logo should be enough? (would also apply to other files)
```suggestion
  <div class="sbb-header-spacer-logo"></div>
```